### PR TITLE
dhewm3: update to 1.5.0

### PR DESCRIPTION
**Pull request by SolitudeSF**

### CODE REVIEW

File: srcpkgs/dhewm3/template [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-distfiles="${homepage}/${pkgname}/archive/${version}.tar.gz"
-checksum=587586142d863ff98051619f0d49c056c5302e919c655a50af69a8293bc3ae08
-build_wrksrc="neo"
+license="GPL-3.0-or-later"
+homepage="https://dhewm3.org/"
+distfiles="https://github.com/dhewm/dhewm3/archive/${version}.tar.gz"
+checksum=fd299c8a2db1ecde15f861e4a883a5d61e5da0c74e26f89fb50b52b978fe8212
+
+case "$XBPS_TARGET_MACHINE" in
+	arm*) makedepends+=" libX11-devel";;
```
Comment (SolitudeSF) [inline srcpkgs/dhewm3/template]: Why is this needed? I have no clue.

Comment (maxice8) [reply srcpkgs/dhewm3/template]: Take it out and see if arm cross still works, if it works then remove it.
